Header header                                   # stamp
gazebo_plugins/ContactState[] states            # array of geom pairs in contact
